source/guides/guides/continuous-integration.md

```diff
@@ -121,10 +121,16 @@ See our {% url 'examples' https://on.cypress.io/docker %} for additional informa
 
 If you are not using one of the above CI providers then make sure your system has these dependencies installed.
 
+## Ubuntu / Debian
 ```shell
 apt-get install xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2
 ```
 
+## RHEL / CentOS
+```shell
+yum install xorg-x11-server-Xvfb gtk2 libnotify GConf2 nss libXScrnSaver alsa-lib
```

Comment:
we should add `libXtst` and a `-y` so you don't have to manually approve them

Reply:
I was just matching package for package with the existing `apt-get` call, which doesn't seem to install `libXtst`.
Curious, is `libXtst` required to run the Electron version as well, or only Chrome Stable?

Regardless, if you want me to add `libXtst`, I'd like to add the equivalent dependency to the `apt-get` command as well.
Any idea what those packages are, or do I need to hunt em down?

Also, I'm fine adding the `-y` if you really want it, but I'm of the opinion that this seems to be code that a user would copy and paste to run. As such, I kinda feel like it's more appropriate to let the user manually accept the resulting dependencies. Thoughts?

Reply:
I believe Ubuntu ships with that dependency already installed. 
Also since must users come from the npm world, where packages are arguably less vetting and verified, and does not ask to confirm every package individually, I think -y is an okay addition

Reply:
And yes, it's required for electron

Reply:
@jennifer-shehane 
any input on this?

1. should `libXtst` be added to both lines?
2. do you agree or disagree with adding `-y` to the commands?

I'm unsure if the `libXtst` dep is an actual requirement or not given that all of my systems have it pre-installed and I can't remove it without breaking other dep chains.

I'm also hesitant to add `-y` to the commands unless the core team likes that approach. @Bkucera's stance about most of the users being `npm` friendly makes sense, but I would counter-point that just because they're `npm` friendly doesn't mean that the shell should suddenly act like `npm`. ;)
Either way, I'd defer to your stance on that one.